src/renderer/containers/Login/LoginContainer.tsx

```diff
@@ -1,28 +1,53 @@
 import * as React from "react";
-import {ReactNode} from "react";
+import {ReactElement, Component} from "react";
 import {Background} from "../../components/Background/Background";
 import {Modal} from "../../components/Modal/Modal";
 import {InputForm} from "../../components/Input/InputForm";
 import {ButtonPrimary, ButtonSecondary} from "../../components/Button/ButtonStandard";
 import {Link} from "react-router-dom";
 import {Routes, OnBoardingRoutes} from "../../constants/routes";
+import {RouteComponentProps} from "react-router";
+import {Notification} from "../../components/Notification/Notification";
+import {Level, Horizontal, Vertical} from "../../components/Notification/NotificationEnums";
+import database from "../../services/db/api/database";
+import {bindActionCreators, Dispatch} from "redux";
+import {connect} from "react-redux";
+import {storeAuthAction,} from "../../actions";
+import {IRootState} from "../../reducers";
 
 interface IState {
     input: string;
+    notification: boolean;
+}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface IOwnProps extends Pick<RouteComponentProps, "history"> {
+}
+interface IInjectedProps{
+    storeAuth: typeof storeAuthAction;
 }
 
-export default class LoginContainer extends React.Component {
+class Login extends Component<
+IOwnProps & IInjectedProps & Pick<IRootState, "register">, IState> {
+
     public state: IState = {
-        input: ""
-    };
-    public handleChange = (e: React.FormEvent<HTMLInputElement>): void => {
-        this.setState({input: e.currentTarget.value});
+        input: "",
+        notification: false,
```

Comment:
notification should be of type `INotification | null`
where 
```
interface INotification {
   title: string;
   message: string;
}
```
And when displaying notification you just check if state.notification is not null
P.S. you can extract notification type to some utils type directory so it can be reused.